package.json

```diff
@@ -18,16 +18,20 @@
   },
   "homepage": "https://github.com/ng-bootstrap/core#readme",
   "devDependencies": {
-    "angular2": "^2.0.0-alpha.46",
+    "@reactivex/rxjs": "5.0.0-alpha.7",
```

Comment:
Not sure it matters, but the package is now provided as rxjs: https://www.npmjs.com/package/rxjs